src/Objects/Job.vala

```diff
@@ -152,16 +152,17 @@ public class Printers.Job : GLib.Object {
         switch (state) {
             case CUPS.IPP.JobState.PENDING:
             case CUPS.IPP.JobState.PROCESSING:
-            case CUPS.IPP.JobState.HELD:
                 return null;
+            case CUPS.IPP.JobState.HELD:
+                return new ThemedIcon ("process-paused");
             case CUPS.IPP.JobState.STOPPED:
-                return new ThemedIcon ("media-playback-pause");
             case CUPS.IPP.JobState.CANCELED:
+                return new ThemedIcon ("process-stop");
             case CUPS.IPP.JobState.ABORTED:
-                return new ThemedIcon ("process-error-symbolic");
+                return new ThemedIcon ("process-error");
             case CUPS.IPP.JobState.COMPLETED:
             default:
```

Comment:
Should not need `default` here as all current cases have been explicitly handled and any new case might need a different icon.  Not blocking for this PR but as this clauses is being touched ...

Reply:
Good call!